src/MultiLocationEncoder.sol

```diff
@@ -1,3 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.13;
 
-contract MultiLocationEncoder {}
+import "./XCMEnums.sol";
+import "./utils/ScaleCodec.sol";
+import "./utils/CompactTypes.sol";
+
+contract MultiLocationEncoder {
+
+    function encodeMultiLocationSameConsensusParachain(
```

Comment:
Is a fairly verbose function name. Given that name of the contract suggests multilocation encoding, could condense the name to `encodeRelativeParachain`.